src/Distortion.cpp

```diff
@@ -327,14 +325,13 @@ void applyDistortion(double ux, double uy, double &dx, double &dy,
   dy = uy;
 
   switch (distortionType) {
-    // Compute undistorted focal plane coordinate given a distorted
-    // focal plane coordinate. This case works by iteratively adding distortion
+    // Compute distorted focal plane coordinate given undistorted
+    // focal plane coordinates. This case works by iteratively adding distortion
     // until the new distorted point, r, undistorts to within a tolerance of the
     // original point, rp.
     case RADIAL: {
       double rp2 = (ux * ux) + (uy * uy);
 
-      if (rp2 > tolerance) {
         double rp = sqrt(rp2);
         // Compute first fractional distortion using rp
         double drOverR =
```

Comment:
I can't highlight the whole thing but can we tab in from lines 335 to 371 since it is no longer in the if statement?